### DISCUSSION TITLE
Welcome to VdMot_Controller Discussions!

### DISCUSSION THREAD

**Discussion by Lenti84** [Lenti84/VdMot_Controller#66] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by harvey637** [comment]
Hi, @ Lenti
I am using homematic and iobroker for my home automation. Lots of stuff with blinds, irrigation, temperatures and so on.
Because of unhandled errors of the original FALMOT-C12 (actuators refuse proper opening at times) I already ordered your system.
Heatingsystem is BOSCH heatpump with ems-esp32 integration, VdMots, PV, lots of temperature readings and raspberrymatic.
I am not quite ready with placement of heating control, now it is on the homematic (wall-thermostat directly coupled falmot-c12), future with iobroker-adapter and MQTT-adapter and rega/rpc connectiong from homematic into your device?
Lots of scripts exist with holidays, homeoffice, geofencing, sunshine detection, in homematic. 

Sure there is something with the assembly of the kit, but still some questions:
- Do I understand correctly, that I need some MQTT on my iobroker? (ans suggestions? MQTT-broker?)
- is it possible to use some adapter like "heizungssteuerung" with your board (percent vales usage possible?)
- is it possible to"connect" a thermostat (HomeMatic and/or HomematicIP) directly (through iobroker/rpc) to one or more valves?
- is it possible to use variables (representing temperatures calculated by homematic eg. with weather forcast/sunshine detection) as source of measured temperature to one or more valves?

I am sure that more questions will arise if starting to integrate your board.
thank you for the work you entered into this project!
cu
Harvey

**Reply by SurfGargano** [reply]
pls see at the iobroker forum
[iobroker VDMot ( in German)](https://forum.iobroker.net/topic/52373/neu-entwickelter-heizungs-controller-f%C3%BCr-elv-hmip-vdmot)

Your questions
Do I understand correctly, that I need some MQTT on my iobroker? (ans suggestions? MQTT-broker?) -> yes, use mqtt server/broker
is it possible to use some adapter like "heizungssteuerung" with your board (percent vales usage possible?) -> yes , you can provide only target and real temperature value to the integrated PI-Controller of VDMot. The Pi Controller will calculate the valves automatically.
is it possible to"connect" a thermostat (HomeMatic and/or HomematicIP) directly (through iobroker/rpc) to one or more valves? -> No, but thru a small script it is possible
is it possible to use variables (representing temperatures calculated by homematic eg. with weather forcast/sunshine detection) as source of measured temperature to one or more valves? ->yes, if you have more than 1 heating circuits in one room you can link valves togehter.
